### PR TITLE
noTask: changed structure of PGRepositories

### PR DESCRIPTION
**Pull request by ivanboika**
1) Tests must be changed and fixed due to new stuff
2) Type of sqlite3 as a type in template is not right or wrong (should be tested)

### CODE REVIEW

File: DataAccess/Public/Include/DataAccess/AbstractRepositoryManager.hpp [hunk 1]
```diff
@@ -153,14 +154,15 @@ namespace DataAccess
...
-            _repositories->registerRepository<IMessagesRepository, MessagesRepository>();
-            _repositories->registerRepository<IRegisterRepository, RegisterRepository>();
-            _repositories->registerRepository<IRepliesRepository, RepliesRepository>();
-            _repositories->registerRepository<IDirectMessageRepository, DirectMessageRepository>();
+            _repositories->registerRepository<IChannelsRepository, PostgreChannelsRepository>();
+            _repositories->registerRepository<ILoginRepository, PostgreLoginRepository>();
+            _repositories->registerRepository<IMessagesRepository, PostgreMessagesRepository>();
+            _repositories->registerRepository<IRegisterRepository, PostgreRegisterRepository>();
+            _repositories->registerRepository<IRepliesRepository, PostgreRepliesRepository>();
+            _repositories->registerRepository<IDirectMessageRepository, PostgreDirectMessageRepository>();
```
Comment (memorisecodead) [inline DataAccess/Public/Include/DataAccess/AbstractRepositoryManager.hpp:165]: Why is Postgre?
We are gonna to common database

**Review by memorisecodead — COMMENTED** [review]
It looks good

**Comment by f1nal3** [comment]
Hey, @ivanboika , since we're on the verge of a new release, please answer the following questions:
1) Do you still work on this PR?
2) How long will it take you to finish it?
3) Can it be reviewed and merged now?
4) How much will we lose if the just discard it?

**Comment by ivanboika** [comment]
@f1nal3 , 
1) I haven't touched this PR for one months or even more
2) Idk, don't quite remember how to fix the gcc/clang errors
3) No, because It works only with windows build, but works properly
4) Not no much, I suppose, PR can be closed without merge

**Pull request closed without merging**